source/elements/oneDAL/source/data_management/accessors.rst

```diff
@@ -17,30 +20,20 @@ Requirements
 Each accessor implementation shall:
 
 1. Define a single :term:`format of the data <Data format>` for the
-   accessor. Every single accessor type shall return and use only one data
-   format.
-
-2. Provide an access to at least one in-memory :txtref:`dataset` implementation
-   (such as :code:`table` or its sub-types).
-
-3. Provide read-only, write-only, or read-write access to the data. If an
-   accessor supports several :txtref:`dataset` implementations to be passed in,
-   it is not necessary for an accessor to support all access modes for every
-   input object. For example, tables shall support a single read-only mode
-   according to their :txtref:`<table> concept` definition.
+   access. Every accessor type shall return and use only one data format.
 
-4. Provide the names for read and write operations following the pattern:
+2. Provide read-only access to the data in the :txtref:`table` types.
 
-   - :code:`pull_*()` for reading
+3. Provide the :code:`pull()` method for obtaining the values from the table.
 
-   - :code:`push_*()` for writing
+4. Be lightweight. Its constructors shall not have computationally intensive
+   operations such data copy, reading, or conversion. These operations shall be
+   performed by method :code:`pull()`. Support of copy- and move- constructors
+   by the accessor is not required since it shall be designed for use in a local
+   scope - directly in a place when it is created.
 
-5. Be lightweight. Its constructors from :txtref:`dataset` implementations
-   shall not have heavy operations such as copy of data, reading, writing, any
-   sort of conversions. These operations shall be performed by heavy operations
-   :code:`pull_*()` and :code:`push_*()`. It is not necessary to have copy- or
-   move- constructors for accessor implementations since it shall be designed
-   for use in a local scope.
+5. The :code:`pull()` method shall avoid data copy and conversion when it is
+   possible to use memory directly from the table.
```

Comment:
when it is possible to return the pointer to the contiguous memory block with requested values in the table?